e2e/pages/StageOne/NamePage.ts

```diff
@@ -76,51 +71,8 @@ export default class NamePage {
     await expect(this.continueButton).toHaveText("Continue");
   }
 
-  // --- CLICK METHODS --- //
-  // check for back button click
-  async clickBack() {
-    await this.backButton.click();
-  }
-
-  async clickContinue() {
-    await this.continueButton.click();
-  }
-
-  // --- FILLING IN FORM METHODS --- //
-  // filling the first name input field
-  async fillFirstName(firstName: string) {
-    await this.firstNameInput.click();
-    await this.firstNameInput.fill(firstName);
-  }
-
-  // filling the last name input field
-  async fillLastName(lastName: string) {
-    await this.lastNameInput.click();
-    await this.lastNameInput.fill(lastName);
-  }
-
-  // --- ERROR CHECKING FORM --- //
-
-  async assertBothErrorMessages() {
-    await expect(this.firstNameError).toBeVisible();
-    await expect(this.firstNameError).toHaveText(/Enter your first name/);
-    await expect(this.lastNameError).toBeVisible();
-    await expect(this.lastNameError).toHaveText(/Enter your last name/);
-  }
-
-  async assetFirstNameError() {
-    await expect(this.lastNameError).toBeHidden();
-    await expect(this.firstNameError).toBeVisible();
-    await expect(this.firstNameError).toHaveText(/Enter your first name/);
-  }
-
-  async assertLastNameError() {
-    await expect(this.firstNameError).toBeHidden();
-    await expect(this.lastNameError).toBeVisible();
-    await expect(this.lastNameError).toHaveText(/Enter your last name/);
-  }
-
-  async assertHideErrorMessages() {
+  // Check no error messages visible before continue button pressed
+  async assertErrorsHidden() {
```

Comment:
Nothing wrong with this, but as mentioned in the utils comments,
If you change the Hidden util method to be generic, and allow it to accept an array like your assertComponentsVisible method.
Then this can become:
  async assertErrorsHidden() {
      await assertComponentsHidden([this.firstNameError, this.lastNameError])
  }